test/product-import.spec.coffee

```diff
@@ -510,6 +510,19 @@ describe 'ProductImport unit tests', ->
         done()
       .catch done
 
+    it 'should throw error in case of too big query', (done) ->
+      exptectedErrorMessage = 'product fetch query size: 10952 bytes, exceeded the supported size, please try with a smaller batch size.'
```

Comment:
What happens if batch size is 1 and its still to long because i have 500 variant skus?